LibraryApp/src/app/AppContainer.js

```diff
@@ -38,7 +39,8 @@ const BooksNavigator = createStackNavigator(
 const TabNavigator = createBottomTabNavigator(
   {
     [Routes.Library]: BooksNavigator,
-    [Routes.EmptyScreen]: EmptyScreen
+    [Routes.EmptyScreen]: EmptyScreen,
+    [Routes.Login]: { screen: Login }
```

Comment:
Why aren't EmptyScreen and Login used the same way?

Reply:
good observation, i don't know either